### DISCUSSION TITLE
New message for insert kill command and addition of which user the attack has been inserted for

### DISCUSSION THREAD

**Discussion by Scientificcc** [c0nD/FluffyBOT#46] [Ideas]
Currently the response when you use the  **insert kill** command is this:
![image](https://user-images.githubusercontent.com/86245936/217880421-376bb531-bb19-485f-8f9b-1572db142694.png)
When you use **admin kill** the response is this:
![image](https://user-images.githubusercontent.com/86245936/217880565-dc703634-903f-4d3c-9cd7-f69999067231.png)

It would be helpful to make the reply for an **insert kill** say that the boss has been killed by using that command and not **admin kill** 
Currently the message for **insert hit** does that and I think it clears up a lot of confusion. 
![image](https://user-images.githubusercontent.com/86245936/217881383-c7db03ca-0483-44c9-9f21-0dee47ce37a8.png)

It could also be nice to have both **insert hit** and **insert kill** tell what user the attack has been inserted for.
Example that could be nice:
![image](https://user-images.githubusercontent.com/86245936/217882966-f825a8e0-994b-4732-9b1e-52759cd81bb9.png)

**Comment by c0nD** [comment]
I think this is a good idea.

Although the example at the end is a bit long for an embed for my personal liking, I can see how adding the person to it adds more clarity!

**Reply by Scientificcc** [reply]
I agree it's a bit wordy. There might be a way to format things differently
